packages/kbn-unified-data-table/src/components/data_table.scss

```diff
@@ -63,6 +63,9 @@
     border-left: 0;
     border-right: 0;
   }
+  .euiDataGridRowCell.euiDataGridRowCell--controlColumn[data-gridcell-column-id='additionalRowControl_menuControl'] .euiDataGridRowCell__content {
```

Comment:
Just curious why we need these styles specifically for the menu action?

Reply:
Its alignment was a bit off for some of the new density settings.